docs/source/autograd.rst

```diff
@@ -11,6 +11,33 @@ Automatic differentiation package - torch.autograd
 
 .. autofunction:: grad
 
+.. _functional-api:
+
+Functional higher level API
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This section contains the higher level API for the autograd that builds on the basic API above.
```

Comment:
nit: since you jump into discussing the jacobian below, why don't you give some examples of what the "high level" autograd functions are, e.g.

"This section contains the higher level API for the autograd that builds on the basic API above and allows you to compute jacobians, hessians, etc.